contrib/gptfdisk/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "gptfdisk"
+pkgver = "1.0.9"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "gmake",
+    "linux-headers",
+    "libuuid-devel",
+    "ncurses-devel",
+    "popt-devel",
+]
+pkgdesc = "Fdisk-like partitioning tool for GPT disks"
+maintainer = "reocat <ng.ct_ml@tuta.io>"
+license = "GPL-2.0-or-later"
+url = "https://sourceforge.net/projects/gptfdisk"
+source = f"$(SOURCEFORGE_SITE)/{pkgname}/{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "dafead2693faeb8e8b97832b23407f6ed5b3219bc1784f482dd855774e2d50c2"
+# no check provided
```

Comment:
this is a lazy way out

cosnidering gptfdisk provides a test suite and it's capable of mostly passing, you should patch out the part that is not relevant for the test environment and let the rest run

Reply:
Should be better now

Reply:
But I can't check it in GitHub CI